packages/components/src/components/Tooltip/index.tsx

```diff
@@ -123,10 +123,12 @@ const Tooltip = ({
     offset = 10,
     cursor = 'help',
     content,
-    openGuide,
+    guideAnchor,
     title,
     ...rest
-}: Props) => {
+}: TooltipProps) => {
+    const [isShown, setIsShown] = useState(false);
```

Comment:
nit: maybe isVisible is little bit more convenient

resolve if think it's good enough

Reply:
It also triggers onShow so I think it's more consistent that way c: